docs/proposals/docker-registry.md

```diff
@@ -0,0 +1,36 @@
+## Improve docker-registry
+
+We need to stop using `linkedBlobStore` from `docker/distribution`. The main purpose for this module is to store information
+about blobs because `docker/distribution` don't have own database. It places information in the filesystem.
+
+Openshift already uses a etcd as a database. All infomation about images/imagestreams stored there. We can hold information
+about the blobs in repository in the database as well. In this case we need store only the blobs's payload in the filesystem
+using our own objects layout. In this case, we get complete control over the blobs.
```

Comment:
We don't need to replace existing layout. We can do just fine with its subset. If we really need to use a different one, let's document why.